src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultTagHelperCompletionService.cs

```diff
@@ -279,10 +277,10 @@ void UpdateCompletions(string tagName, TagHelperDescriptor possibleDescriptor)
 
             var binding = _tagHelperFactsService.GetTagHelperBinding(
                 completionContext.DocumentContext,
-                completionContext.ContainingTagName,
-                completionContext.Attributes,
                 completionContext.ContainingParentTagName,
-                completionContext.ContainingParentIsTagHelper);
+                completionContext.Attributes,
+                parentTag: null,
```

Comment:
Why `null` now?

Reply:
Null because we don't have it. Now that we're looking at `ContainingParentTagName` for the main name to bind against we would have to do something like `ContainingGrandparentTagName` in order to put something else on that field.

I can go add that property to `ElementCompletionContext` (and start populating it), but it wasn't clear to me what value it would provided so I erred on the side of not doing it.

Reply:
If it doesn't break anything pre-existing then the scenario seems fine to me to add